### DISCUSSION TITLE
MSVC can't open include file <Eigen/Core>

### DISCUSSION THREAD

**Discussion by patrikhuber** [compiler-explorer/compiler-explorer#7188] [General]
Hi,

This feels like a stupid question, I'm probably just missing something dumb. The line `#include <Eigen/Core>` results in a compiler error on CE with MSVC x64 19.latest (and 19.40), `<source>(1): fatal error C1083: Cannot open include file: 'Eigen/Core': No such file or directory`. The code works fine on gcc.
I did go to the Libraries options dialogue and selected Eigen for both the gcc and the MSVC compiler window.

I'm sure I've done this in the past and it worked... but I can't figure it out today.

What could I be doing wrong?

**Comment by mattgodbolt** [comment]
Unfortunately we don't support libraries on Windows builds currently. This isn't well documented, sorry for you tripping over this. We did used to support this, but lost it in the transition from MS hosted compilers to our own.
It's on the list of things to do...

**Reply by patrikhuber** [reply]
Thank you for the reply Matt! I see, that makes sense. FYI, the Libraries window says on MSVC, vcpkg is used - this misled me:

![image](https://github.com/user-attachments/assets/9867b552-7ed3-4c90-be3b-a037c896b30b)

I also just saw https://github.com/compiler-explorer/compiler-explorer/issues/7194 now, and I can see that libraries on MSVC were supported in the past, but are not anymore at the moment. Hope this'll come back.

**Comment by mattgodbolt** [comment]
Wow! Great point, thank you for noting the wrong text in the Libraries dropdown, will update immediately!

**Reply by mattgodbolt** [reply]
PR #7198
